client/src/components/Onboarding/OnboardingAdmin.module.scss

```diff
@@ -76,44 +82,63 @@ tbody {
 .root > tbody > tr > td {
   padding: 0 0 0 0;
 }
+*/
 
-.name {
+.user > td {
+  /* Difference between tall unwrapped cell height and basic text cell height */
+  --greatest-height-diff: 4.78px; /* value obtained form live render */
+  /* FAQ: We want only the top space, not the top & bottom space combined */
+  --vertical-offset: calc(var(--greatest-height-diff) / 2);
+}
+.user > td:not(.has-wrappable-content) {
+  /* Tweak `vertical-align`'s `top` to look like `middle` when nothing wraps */
+  padding-top: calc(var(--cell-horizontal-padding) + var(--vertical-offset));
   vertical-align: top;
 }
-
-.user > td > div {
-  height: 32px;
-  padding-left: var(--cell-horizontal-padding);
-  display: flex;
-  align-items: center;
+.user > td:not(.has-wrappable-content):not(.status) {
+  @include truncate-with-ellipsis;
 }
-
-.user > td > div:nth-child(even):not(.staffwait) {
+.user:nth-child(even) > td:not(.staffwait):not(.name) {
   background-color: #C6C6C61A;
 }
 
-.status {
-  line-height: 19px;
+/* HACK: This selector has knowledge of sibling component's internal markup */
+/* FAQ: Because we cannot pass `className`to <Pill> via <OnboardingStatus> */
+/* FAQ: Because <OnboardingStatus> has a superfluous root element */
+.status > span > span /* i.e. <OnboardingStatus>'s <Pill> */ {
+  white-space: nowrap;
 }
 
-.reset > * {
-  padding-left: 8px;
-  padding-right: 8px;
+.reset {
+  /* … */
 }
```

Comment:
**Explanation**.

- Removed padding so space between was even less—just a single text space.
- Retained class so I do not need to remove it from HTML. (I am trying to limit diff to prevent refactor.)